packages/clerk-js/src/ui/elements/Header.tsx

```diff
@@ -19,12 +19,18 @@ const Root = React.memo(
       <Col
         ref={ref}
         elementDescriptor={descriptors.header}
-        gap={6}
+        gap={gap}
         sx={sx}
         {...rest}
       >
         {logoIsVisible && <ApplicationLogo />}
-        <Col gap={1}>{children}</Col>
+        <Col
+          gap={1}
+          sx={sx}
```

Comment:
So here, we pass the same `sx` prop to 2 different elements. This will cause issues.

Reply:
I was think the same I will have to revert that.

Reply:
Fixed!